app/helpers/vacancies_helper.rb

```diff
@@ -22,6 +22,14 @@ def link_to_sort_by(title, column:, order:, sort:)
             class: "sortby--#{order}#{active_class || ''}"
   end
 
+  def pluralize_vacancy_count(count, i18n_id, plural_i18n_id = nil)
+    if count == 1
+      I18n.t(i18n_id, count: count)
+    else
+      I18n.t(plural_i18n_id || (i18n_id + '_plural'), count: count)
+    end
+  end
+
```

Comment:
This count is specific to vacancies so it would make more sense to be in the VacanciesPresenter, with a beteer name, as the purpose of the presenter is to remove unecessary noise from the views e.g. `pluralize_vacancy_count` could be `vacancies.total_count`

Reply:
As mentioned I cannot use the built in `total_count` method, it returns everything regardless of filters. 

The source of this bug is that the pagination is half handled inside the presenter, half handled by the controller. The controller is in currently responsible for the reading of data and the pagination. The presenter only every gets those paginated vacancies, it doesn't have access to the total value.